### PR TITLE
MB-16587: update move submission email

### PR DESCRIPTION
**Pull request by ronaktruss**
## Summary

This updates the email notification that is sent out when a move is submitted. 

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. To make the email sending works locally follow the steps here: https://dp3.atlassian.net/wiki/spaces/MT/pages/1467973894/How+to+test+sending+email+locally
2. Create a new user and make sure to use your email address as the contact info.
3. Create and submit an HHG shipment.

## Screenshots

Screenshot 2023-08-10 at 2 37 00 PM

Screenshot 2023-08-10 at 2 55 46 PM

Screenshot 2023-08-10 at 2 45 55 PM

### CODE REVIEW

**Comment by khayenis** [comment]
@ronaktruss 

- Can we remove "your nearest transportation office" so it's not redundant? 

"To change any information about your move, or to add or cancel shipments, you should contact ~~your nearest transportation office at~~ (555) 555-5555 or visit your <local transportation office>."

- I'm struggling to figure out who to test sending an email locally even with the instructions 👀

**Comment by ronaktruss** [comment]
@khayenis Yeah I can make that change. 

Oops, those instructions were meant for other devs. I could make these changes available in exp but that would still require you to go through creating all of these moves to initiate the emails. Let me get some screen shots of the major variations of the email and then attach those to the description. 

If you just want to verify the text the contents are coming from this file: https://github.com/transcom/mymove/blob/9c9b13fddf3aa64bca861118a7068897ddbd8db4/pkg/assets/notifications/templates/move_submitted_template.html. Note: There are some additional curly braces are are used for the conditional logic.

**Review by mmayo-truss — APPROVED** [review]
Looks good to me! Design should probably sign off on this as well.

File: pkg/notifications/move_submitted.go [hunk 1]
```diff
@@ -62,9 +68,9 @@ func (m MoveSubmitted) emails(appCtx appcontext.AppContext) ([]emailContent, err
 		return emails, err
 	}
 
-	var originDutyLocation, originDutyLocationPhoneLine *string
+	var originDutyLocationName, originDutyLocationPhoneLine *string
```
Comment (kctruss) [inline pkg/notifications/move_submitted.go:71]: **praise:** I appreciate the improved context naming here.‏

**Review by kctruss — APPROVED** [review]
🌟 Looks great to me!

**Comment by ronaktruss** [comment]
@khayenis I've made those changes and updated the screenshots. (Ignore the purple text and `...` they are just artifacts of gmail sending these emails consecutively.)

The updated text can be seen here https://github.com/transcom/mymove/blob/ea4fcefac5fb96efc2c9c0b32cc43e5af013e735/pkg/assets/notifications/templates/move_submitted_template.html

**Comment by khayenis** [comment]
LGTM, thanks for the screenshots @mmayo-truss

**Review by khayenis — APPROVED** [review]

**Pull request merged**